plugin/core/src/test/groovy/com/novoda/gradle/release/ReleasePluginTest.groovy

```diff
@@ -62,28 +62,28 @@ class ReleasePluginTest {
 
     @Test
     void shouldBuildLibrary() {
-        GradleTruth.assumeThat(result).isSuccess()
+        GradleTruth.assumeThat(configuration.expectedBuildSuccess).isTrue()
```

Comment:
The precondition here was wrong: it was looking at the actual outcome of the build instead of considering what was the expectation for it 🙈 
The idea was that if we expect the build to fail there's no reason to go and check all the tasks and their output, so it's safe to skip those tests.